Home_Work_1/task2.py

```diff
@@ -0,0 +1,21 @@
+# Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: «Число является простым, если делится нацело только на единицу
+# и на себя». Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+LOWER_LIMIT = 0
+UPPER_LIMIT = 100000
+
+while True:
+    number = int(input('Введите число: '))
+    if number > LOWER_LIMIT and number <= UPPER_LIMIT:
+        count = 0
+        for i in range (2, number):
+            if number % i == 0:
+                count = i
+                print('Вы ввели составное число')
+                break
+        if count == 0:
+            print('Вы ввели простое число')
```

Comment:
Мне не до конца понятно зачем вам переменная `count`, когда можно обойтись без нее, сделав так
```suggestion
        for i in range (2, number):
            if number % i == 0:
                print('Вы ввели составное число')
                break
        else:
            print('Вы ввели простое число')
```